policies/trademark-policy.md

```diff
@@ -1,172 +1,325 @@
+
+- If a company makes "Foobar" software and a third party offers training called "Foobar Certification," a person is likely to believe, wrongly, that the certification is being offered by the makers of Foobar software. The third party has likely misled consumers about the source of its training and is a trademark infringer.
+
+### What is "nominative" use?
+
+So-called "nominative use" (or "nominative fair use"), which is the name of the doctrine under U.S. trademark law, allows the use of another's trademark where it is necessary for understanding.
+Other countries' trademark laws also have similar provisions.
+
+For example, a car repair shop that specializes in a particular brand of automobile, VW for example, must be allowed to say that they repair VW cars.
+Here is what you should consider when deciding whether your use of a trademark is a nominative fair use:
+
+- Can you identify the product or service in question without using the trademark;
+- Are you avoiding a likelihood of confusion in the way that you have used the trademark; and
+- Have you used only as much as is necessary to identify the product or service.
+
+With our "Foobar Certification" example above, the person offering the certification would be allowed to say, under the nominative fair use doctrine, that she is offering "Maude's Certification for Foobar software."
+
+It is almost never the case that using a Logo will be a nominative fair use since it will be a rare case where the logo is needed for strictly informational purposes.
+
+## Proper trademark use
+
+These rules hold true for all trademarks, not just ours, so you should follow them for our Marks as well as anyone else's.
+
+### Use of trademarks in text
+
+- Always distinguish trademarks from surrounding text with at least initial capital letters or in all capital letters.
+    - Unacceptable: hibernate
+    - Acceptable: Hibernate, HIBERNATE
+
+- Always use trademarks in their exact form with the correct spelling, neither abbreviated, hyphenated, or combined with any other word or words.
+    - Unacceptable: Open Rewrite, open rewrite, open-rewrite, openrewrite
+    - Acceptable: OpenRewrite
+    - See our [Trademark List][] to verify exactly how the marks should appear.
+
+- Don't pluralize a trademark.
+    - Unacceptable: I have seventeen JBangs running in my lab.
+    - Acceptable: I have seventeen JBang catalogs running in my lab.
+
+- Don't use "a" or "the" to refer to an instance of the trademark.
+    - Unacceptable: I put a JBang on my mom's computer.
+    - Acceptable: I put a JBang catalog on my mom's computer.
+
+- Don't use a trademark as a possessive.
+   The following noun should be used in possessive form or the sentence reworded so there is no possessive.
+   An exception to this can be when CF is used to refer to the Foundation‘s name.
+
+    - Unacceptable: OpenRewrite's unique properties ...
```

Comment:
Also overly prescriptive. I'm pretty sure we've written this in the possessive many times in written publications.

Reply:
These are general practices for preserving the value of the mark over time. 

This is best practice according to the [International Trademark Association](https://www.inta.org/fact-sheets/trademark-use/).

The guidelines we list in this section come from that publication, in essence, but are placed in context with our Projects and reference our list of trademarks. 

I will add a reference to those materials at the top of the section.